int32 sourceNumber
int32 mergedNumber
geometry_msgs/Point sourceCentroid
geometry_msgs/Point mergedCentroid
